src/otx/algo/segmentation/segnext.py

```diff
@@ -15,4 +15,9 @@ class SegNext(MMSegCompatibleModel):
     def __init__(self, num_classes: int, variant: Literal["b", "s", "t"]) -> None:
         model_name = f"segnext_{variant}"
         config = read_mmconfig(model_name=model_name)
+        self.image_size = config["data_preprocessor"].get("size", (512, 512))
```

Comment:
Is 512 the final resolution for SegNext?
Is it aligned with OTX torch test?

Reply:
Actually, 512 512 looks better for both models, but I need to test it more. Let's keep 544 for now